cli/src/cluster_query.rs

```diff
@@ -2181,6 +2181,9 @@ pub fn process_calculate_rent(
     data_length: usize,
     use_lamports_unit: bool,
 ) -> ProcessResult {
+    if data_length > 10_000_000 {
```

Comment:
Can you use https://github.com/anza-xyz/agave/blob/1189055c98672d3d439cd1574d650c49dbf38e77/sdk/program/src/system_instruction.rs#L85 instead of this value?